### PR TITLE
update VENDOR_ID, PRODUCT_ID, and DEVICE_VER for Duck boards

### PR DESCRIPTION
**Pull request by mechmerlin**
## Description

Update IDs associated with Duck boards to uniquely identify them for future VIA support. Copy pasta all around has resulted in almost all QMK supported boards with the same vendor and product ids. 

## Types of changes

- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [x] Enhancement/Optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/Layout/Userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by this PR

* 

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by drashna** [comment]
Does the normal firmware for the Duck boards use these ID's?

File: keyboards/duck/octagon/v1/config.h [hunk 1]
```diff
@@ -21,9 +21,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
+#define VENDOR_ID       0x444B // Duck ("DK")
 #define PRODUCT_ID      0x6050
```
Comment (noroadsleft) [inline keyboards/duck/octagon/v1/config.h]: Any reason this isn't `0x3847` like the Octagon v2?

Comment (mechmerlin) [reply keyboards/duck/octagon/v1/config.h]: The stars in the night illuminated a portion of the fifth galaxy. This sent tremors across the mortal planes affecting keyboards far and wide. 

tl;dr: I'm blind.

**Comment by mechmerlin** [comment]
> Does the normal firmware for the Duck boards use these ID's?

I've actually never checked, and I'm not touching o2d with a ten foot pole ever again.

**Comment by drashna** [comment]
Is VIA officially supporting them? 

And are these in use?

**Comment by Wilba6582** [comment]
FWIW VIA Configurator doesn't use `DEVICE_VER`, only `VENDOR_ID` and `PRODUCT_ID`. There is the assumption that devices with that unique pair will have the same layout and matrix (rows/columns). So, don't use `DEVICE_VER` as a way to differentiate between two versions of the same device; encode that into `PRODUCT_ID`. Note if two versions of a device just differ by PCB layout, I/O pin usage, slightly different firmware code, etc. then it's OK for them to have the same `VENDOR_ID` and `PRODUCT_ID`.

**Comment by mechmerlin** [comment]
Unsure how many other versions if any the other Duck boards will have. Incorporated Wilba's suggestions into the Octagon V1/V2 for the time being.
